include/zephyr/net/wifi_mgmt.h

```diff
@@ -359,6 +359,8 @@ struct wifi_connect_req_params {
 	enum wifi_security_type security;
 	/** MFP options */
 	enum wifi_mfp_options mfp;
+	/** BSSID */
+	uint8_t *bssid;
```

Comment:
should we make this an array as the size is known at compile time?

Reply:
This way we avoid an extra `memcpy` which may be required if we use a static array.

Reply:
It might causes issues with memory management, no? In `wifi_shell.c` you are passing `optarg`, and once the shell function returns it will be freed (as `argv` will be out of scope).

Reply:
Done.